scripts/ansible/roles/avalanche_base/tasks/ubuntu.yml

```diff
@@ -0,0 +1,8 @@
+- name: Install Avalanche dependencies
+  become: true
+  apt:
+    name:
+      # Staking key management
+      - openssl
+      - python3-cryptography
```

Comment:
I think a bare-bones installation of `ubuntu` doesn't have `gcc` installed and I believe I have had issues building without it. I usually install `build-essential` in such cases.

Reply:
Not sure what would be the equivalent for the other platforms

Reply:
I've tested to confirm, build-essential is not needed.